include/utility/vpd_specific_utility.hpp

```diff
@@ -466,5 +466,89 @@ inline bool findCcinInVpd(const nlohmann::json& i_JsonObject,
     logging::logMessage("VPD type not supported. Can't find CCIN");
     return false;
 }
+
+/**
+ * @brief API to clear VPD of the FRU on DBus
+ *
+ * This API deletes the VPD for particular interfaces of a FRU on DBus.
+ *
+ * @param[in] i_objectPath - DBus object path of the FRU.
+ * @param[in] i_interfaceMap - Interface and its properties map.
+ */
+inline void clearVpdOnRemoval(const std::string& i_objectPath,
+                              types::InterfaceMap& i_interfaceMap)
```

Comment:
`io_interfaceMap` ? Since `i_interfaceMap` is an input/output parameter.

Reply:
Updated